src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -112,6 +111,15 @@
         return reportFoldersAndFiles;
     }
 
+    /**
+     * Copies the template file to the specified {@code outputPath} for the repo report to be generated
+     * @throws IOException if {@code is} refers to an invalid path.
```

Comment:
For header comments, especially static methods, we do not quote internal variables in javadoc because its meant to be hidden/abstract. So in this case, you can be more explicit about it, e.g. if the template resource is not found.

Reply:
Thanks for the feedback. Just changed it.